hax/helper/configure.py

```diff
@@ -135,9 +136,12 @@ def get_join_ip() -> str:
         ]),
                           env=self._get_pythonic_env())
 
+        machine_id = Conf.machine_id
+
         self.executor.run(Program([
             'update-consul-conf', '--conf-dir', f'{self.conf_dir}',
-            '--kv-file', f'{self.conf_dir}/consul-kv.json'
+            '--kv-file', f'{self.conf_dir}/consul-kv.json',
+            '--uuid', machine_id
```

Comment:
@SwapnilGaonkar7, in-order for --uuid to be optional, we need to add this optional to configure.py as well.